BuildFile.xml

```diff
@@ -1,2 +1,3 @@
 <use name="root"/>
+<use name="boost"/>
```

Comment:
If possible, I would try to avoid additional dependencies.

Reply:
Done.